sass/base/_btn.scss

```diff
@@ -15,19 +15,14 @@ Markup: html/btn.html
 Style guide: buttons
 */
 
-@mixin _white_space_reset_mixin() {
-  padding-top: 0.5em;
-  padding-bottom: 0.5em;
-  line-height: 1.667;
-  white-space: normal;
-}
-
 .btn {
   @extend .hidden-print;
   min-height: 2.75em;
   margin-bottom: 1em;
-  padding: 0.75em 1em;
-  line-height: 1;
+  padding: 0.5em 1em;
+  line-height: 1.5;
+  white-space: normal; // Override Bootstrap white-space: nowrap;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/14326885/25392149/30baad54-29d8-11e7-9003-2b918c3ac0a7.png)

This is, i think, one of the reasons why this behavior by bootstrap has never been overridden.
But honestly, i could think of other situation where i'm not entirely sure they would be happy about the results.
Anyway, try to see if this issue with the external icon (i'm the first enemy of this icon... ) can be tackled in a decent way, in case we might then customize a little bit more the examples in the styleguide to show some "critical scenarios" for them to see the downsizes of this operation.